app/styles/modules/_avatar.scss

```diff
@@ -142,56 +142,49 @@
 }
 
 .avatar-change {
-  .gravatar-logo {
-    background: image-url('gravatar.svg') no-repeat top center;
-    background-size: contain;
-    /* This reduces the size of the gravatar logo to match the other icons */
-    font-size: 30px;
-
-    /* Mimic the style of the fontello icons */
```

Comment:
It looks like we use a few still.
```
app/scripts/templates/partial/close-button.mustache
1:<a href="#" id="close" class="icon-cancel-circled"></a>

app/scripts/templates/settings/avatar_crop.mustache
15:        <span class="zoom-out icon-zoom-out"></span>
17:        <span class="zoom-in icon-zoom-in"></span>
18:        <button class="rotate icon-cw"></button>
```

I'll see if it is easy to generate a set of fonts with just those icons.